mmhuman3d/core/constants.py

```diff
@@ -0,0 +1,271 @@
+import smplx.joint_names as smplx_joint_name
```

Comment:
MMHuman3d keeps conventions under `mmhuman3d/core/conventions/keypoints_mapping`. Please try to kindly merge them.

Reply:
Done.